src/components/pill-selector-dropdown/stories/PillSelectorDropdown.notes.md

```diff
@@ -0,0 +1,5 @@
+This component renders a selector dropdown with pills for selected options.
+It handles keyboard events for accessibility and controls the input value.
+The children should be `DatalistItem` components for the dropdown options.
+
+`import PillSelectorDropdown from 'src/components/pill-selector-dropdown';`
```

Comment:
This is lame, but no other way to have auto imports
https://github.com/storybookjs/storybook/issues/7772

Also, using CSF it DOES NOT generate a "copy" button like below 😠 
<img width="806" alt="Screen Shot 2019-11-13 at 4 15 55 PM" src="https://user-images.githubusercontent.com/1075325/68903297-9251b480-06ef-11ea-96b7-0c47e7e083dd.png">